### PR TITLE
Add a tvOS build workflow

### PR DESCRIPTION
**Pull request by DellaBitta**
First pass at a workflow to build the Unity SDK for tvOS platforms.

***

Manaully executed the python scripts which would be invoked by the workflow.

***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [X] Other, such as a build process or documentation change.
***

### CODE REVIEW

File: .github/workflows/build_tvos.yml
Comment (cynthiajoan) [inline .github/workflows/build_tvos.yml:5]: I think we don't need this for subworkflow, we can keep it for test purpose, but once it's hooked with the build_starter.yml, we need to remove this block.

File: .github/workflows/build_tvos.yml
Comment (cynthiajoan) [inline .github/workflows/build_tvos.yml:61]: echo to the previous comment, this would not take effect if the workflow start with dispatch.

File: .github/workflows/build_tvos.yml
Comment (cynthiajoan) [inline .github/workflows/build_tvos.yml:143]: is the build_zip python script update to create tvos_unity folder? Not sure i've seen the related PR yet.

File: .github/workflows/build_tvos.yml
Comment (DellaBitta) [reply .github/workflows/build_tvos.yml:5]: Yes, that's for me to iterate on.

File: .github/workflows/build_tvos.yml
Comment (DellaBitta) [reply .github/workflows/build_tvos.yml:143]: Yep!

File: .github/workflows/build_tvos.yml
Comment (DellaBitta) [reply .github/workflows/build_tvos.yml:61]: Confused by this comment. Both the dispatch and the trigger have these inputs, no?

File: .github/workflows/build_tvos.yml
Comment (cynthiajoan) [reply .github/workflows/build_tvos.yml:143]: Great, if that's the case the check needs to update to if [ -f tvos_unity/*.zip ]

File: .github/workflows/build_tvos.yml
Comment (cynthiajoan) [reply .github/workflows/build_tvos.yml:61]: My experience is the workflow_dispatch use another parameter name for the input, needs to be events.inputs something like that. workflow_call is using "inputs" directly. examples can be found build_starter.yml, - id: set_outputs step.

**Review by cynthiajoan — APPROVED** [review]
LGTM, just prefer to have the check build success or not step's path corrected from ios_unity to tvos_uniy

**Comment by DellaBitta** [comment]
Oh yes, I missed the `ios_unity` -> `tvos_unity`  in the bash if statement. Will fix before merge. Thanks!

**Review by a-maurice — APPROVED** [review]
Approved assuming the ios -> tvos fix that was already called out.

File: .github/workflows/build_tvos.yml
Comment (DellaBitta) [reply .github/workflows/build_tvos.yml:61]: I think it should be ok, based on this reference:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

**Pull request merged**
